oximeter/db/src/model.rs

```diff
@@ -5,17 +5,21 @@
 //! Models for timeseries data in ClickHouse
```

Comment:
The changes in this file are almost exclusively moving types from here to the crate root.